wordpress-mobile/WordPress-iOS
# UL&S: Navigate to a new, blank UnifiedSignUpViewController

Ref. #14009 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/348

### To test:
1. Enable the `.unifiedSignup` feature flag
2. `rake dependencies`
3. Build and run
4. Sign up for WordPress.com > Sign up with Email

Expected: 
- [x] The bottom bar shows a consistent color (not light gray!). 
- [x] The status bar changes color depending on mode.
- [x] The nav bar has a title and two functioning nav bar item buttons.
- [x] The pink button says "Continue". (No implementation yet).

### Screenshots
**iPhone 11 Pro**
| Light Mode | Dark Mode |
| --- | --- |
| <a href="https://user-images.githubusercontent.com/1062444/88963889-23f95600-d26e-11ea-940e-acec5d68aaeb.png"><img src="https://user-images.githubusercontent.com/1062444/88963889-23f95600-d26e-11ea-940e-acec5d68aaeb.png" width="350" /></a> | <a href="https://user-images.githubusercontent.com/1062444/88963929-2fe51800-d26e-11ea-878b-f8807980cfce.png"><img src="https://user-images.githubusercontent.com/1062444/88963929-2fe51800-d26e-11ea-878b-f8807980cfce.png" width="350" /></a> |

### Do not merge until:
- this PR points to a published Auth version ✔️ 
- this PR points to the `develop` branch (we're having Peril issues, which prevented me from merging the feature flag PR before this PR) ✔️ 

PR submission checklist:
